<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuCreation</class>
 <widget class="QMainWindow" name="MenuCreation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loadSim">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Charger une simulation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadModele">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Charger un modèle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createSim">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Créer une simulation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>-10</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cellul'UT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
